docusaurus/docs/reactnative/basics/offline_support.mdx

```diff
@@ -8,7 +8,8 @@ React Native Chat SDK provides OOTB support for offline mode, which means user w
 with chat even when the network is off. This is an opt-in feature that needs to be explicitly enabled on application
 level.
 
-> Offline support is currently not implemented for Expo package (stream-chat-expo).
+> Offline support is currently not implemented for Expo package (stream-chat-expo). However, you can get things working via
+the [expo-dev-client](https://docs.expo.dev/development/getting-started/) library and adding [config plugins](https://docs.expo.dev/guides/config-plugins/)
```

Comment:
```suggestion
> Integration with Expo is possible using the [expo-dev-client](https://docs.expo.dev/development/getting-started/) library. Offline support cannot be used in the "Expo Go" app, because [it requires custom native code](https://docs.expo.io/workflow/customizing/).
```

inspired by https://rnfirebase.io/#expo. Also we do not need to implement in stream-chat-expo. That was a bit confusing.